packages/utils/src/rule-utils/index.ts

```diff
@@ -399,8 +385,8 @@ const createRuleUtilsCreator = (
           ? evalPointer(parentPointer, file.contents)
           : undefined
       },
-      report(items: ReportItem | ReportItem[]): void {
-        addReportsToViolations(items, violations, assistant, rule, pointers, ignoredObjects)
+      report(message: string, objects: SketchFileObject[] = []): void {
```

Comment:
Since the vast majority of the rules will flag a single object, can the API be designed to use variadic argument instead, see [demo](https://www.typescriptlang.org/play/index.html?ssl=1&ssc=1&pln=8&pc=2#code/E4UwDg9sAuAUAGALEAbFF4BoAE8DuUKAJvAJQBQ5KI02AtgM4DmD2AvNgNrwBmEGOeACMAhsHgBdbOVCQYsAHRLGLCuR4BXAHYBjaAEsIW7LKhwlCuiAYMRTawC5sDaMH1amnCaWwBvctiB9Na29gwKfMAAoiI6iLCwAG4iKDj6RAAePmwAfNg6RgwQ1AroTAgAMhBMTgAkvukZAL7YgMgE2PXJKE1kFE1AA)? 🤔 

```ts
report(`hello`, `world`)
// Log: 0 – hello
// Log: 1 – world

let msgs = [`foo`, `bar`] 
report(...msgs)
// Log: 0 – foo
// Log: 1 – bar

function report(...messages: string[]) {
    messages.forEach((val, idx) => console.log(`Log: ${idx} – ${val}`))
}
```

```suggestion
      report(message: string, ...objects: SketchFileObject[] = []): void {
```

Reply:
You're suggestion has merit, I'll look into it ... 🤔